cmake/CMakeLists.txt

```diff
@@ -355,30 +355,23 @@ foreach(SIMPLE_LIB REAX MEAM POEMS USER-ATC USER-AWPMD USER-COLVARS USER-H5MD
         add_library(${PKG_LIB} STATIC ${${PKG_LIB}_SOURCES})
       endif()
       list(APPEND LAMMPS_LINK_LIBS ${PKG_LIB})
-    endif()
-    foreach(PKG_SRC ${${SIMPLE_LIB}_SOURCES})
       if(PKG_LIB STREQUAL awpmd)
-        set_property(SOURCE ${PKG_SRC} APPEND PROPERTY COMPILE_FLAGS -I${LAMMPS_LIB_SOURCE_DIR}/${PKG_LIB}/systems/interact)
-        set_property(SOURCE ${PKG_SRC} APPEND APPEND_STRING PROPERTY COMPILE_FLAGS " -I${LAMMPS_LIB_SOURCE_DIR}/${PKG_LIB}/ivutils/include")
+        target_include_directories(awpmd PUBLIC ${LAMMPS_LIB_SOURCE_DIR}/awpmd/systems/interact ${LAMMPS_LIB_SOURCE_DIR}/awpmd/ivutils/include)
```

Comment:
I think that is not a good idea. If you only have one main target (i.e. `liblammps`) `target_include_directories()` and `include_directories()` are basically doing the same thing.
I was looking for a way to only add the libraries include dirs to the source, which actually need them...

Reply:
src/USER-AWPMD/pair_awpmd_cut.cpp:
```c++
#include "TCP/wpmd_split.h"
```

Reply:
Sorry, my point was a different one. Don't we do this PR to minimize possibility of header and include directory conflicts? 

For this reason I only add the include path to the sources (with `set_property(SOURCE...)`, which need it and not to all of `liblammps`, which is what `target_include_directories()` (or `include_directories()`) does.

Reply:
Yes. Up to the point that it mimics the situation in the old build system. Nothing more, nothing less.